hermes/crates/cbork/abnf-parser/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+// cspell: words apskhem ABNF abnf Naur
```

Comment:
```suggestion
// cspell: words Naur
```
add yourself to the project words